README.md

```diff
@@ -267,6 +267,26 @@ Headers: "<http response headers>"
 Result: "<response returned by service>"
 ```
 
+### Transaction IDs
+Every SDK call will return a response which will contain a transaction ID, accessible via the `x-global-transaction-id` header. This transaction ID is useful for troubleshooting and accessing relevant logs from your service instance.
+
+
+```ruby
+require "ibm_watson"
+include IBMWatson
+
+assistant = AssistantV1.new(
+  authenticator: "<authenticator>"
+  version: "2017-04-21"
+)
+
+response = assistant.headers(
+  "Custom-Header" => "custom_value"
+  ).list_workspaces
+
+p "Global Transaction Id: #{response.headers["X-Global-Transaction-Id"]}"
```

Comment:
Is this the location of the global transaction id in the case of an error as well?